Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -2274,10 +2274,14 @@ bool CStaticFunctionDefinitions::SetPedAnimationProgress(CClientEntity& Entity,
                 pAnimAssociation->SetCurrentProgress(fProgress);
                 return true;
             }
+
+            Ped.m_AnimationCache.progress = fProgress;
         }
         else
         {
             Ped.KillAnimation();
+            Ped.m_AnimationCache.progress = 0.0f;
```

Comment:
Shouldn't this be inside *KillAnimation*?

Reply:
Actually, it's unnecessary since RunNamedAnimation will overwrite animCache anyway